Flaky test: SimpleSharedMacTestSuiteIT

SimpleSharedMacTestSuiteIT is not finding any tests to execute when
run with the sunny profile.  The following stack trace occurs for me
when I run: `mvn clean verify -Psunny`.

```
[ERROR] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 15.90 s <<< FAILURE! -- in org.apache.accumulo.suites.SimpleSharedMacTestSuiteIT
[ERROR] org.apache.accumulo.suites.SimpleSharedMacTestSuiteIT -- Time elapsed: 15.90 s <<< ERROR!
org.junit.platform.suite.engine.NoTestsDiscoveredException: Suite [org.apache.accumulo.suites.SimpleSharedMacTestSuiteIT] did not discover any tests
	at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.accept(ForEachOps.java:183)
	at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:197)
	at java.base/java.util.Iterator.forEachRemaining(Iterator.java:133)
	at java.base/java.util.Spliterators$IteratorSpliterator.forEachRemaining(Spliterators.java:1845)
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:509)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:499)
	at java.base/java.util.stream.ForEachOps$ForEachOp.evaluateSequential(ForEachOps.java:150)
	at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(ForEachOps.java:173)
	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.base/java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:596)
```

However, when I run the following, the test passes: `mvn clean verify -Dit.test=SimpleSharedMacTestSuiteIT -Dtest=foo -Dspotbugs.skip`

It looks like the sunny profile stopped working after commit `f70a12cced92eee4dbbb0a801c4cd9f982227d9a`.

--- a/test/src/main/java/org/apache/accumulo/test/ComprehensiveTableOperationsIT_SimpleSuite.java
+++ b/test/src/main/java/org/apache/accumulo/test/ComprehensiveTableOperationsIT_SimpleSuite.java
@@ -18,6 +18,7 @@
  */
 package org.apache.accumulo.test;
 
+import static org.apache.accumulo.harness.AccumuloITBase.SUNNY_DAY;
 import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertFalse;
@@ -91,6 +92,7 @@ import org.junit.jupiter.api.AfterAll;
 import org.junit.jupiter.api.AfterEach;
 import org.junit.jupiter.api.BeforeAll;
 import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Tag;
 import org.junit.jupiter.api.Test;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
@@ -103,6 +105,7 @@ import com.google.common.net.HostAndPort;
  * avoiding duplicating existing testing. This does not test for edge cases, but rather tests for
  * basic expected functionality of all table operations against user tables and all system tables.
  */
+@Tag(SUNNY_DAY)
 public class ComprehensiveTableOperationsIT_SimpleSuite extends SharedMiniClusterBase {
   private static final Logger log =
       LoggerFactory.getLogger(ComprehensiveTableOperationsIT_SimpleSuite.class);

